cfme/rest/__init__.py

```diff
@@ -0,0 +1,28 @@
+import fauxfactory
+from utils.wait import wait_for
+
+
+def service_catalogs(request, rest_api):
```

Comment:
It is supposed to be imported and used as a fixture. So I would say not call it like `add_random_service_catalogs` since you cannot use it stand-alone.